charts/spire/README.md

```diff
@@ -73,7 +73,13 @@ kubectl delete crds clusterfederatedtrustdomains.spire.spiffe.io clusterspiffeid
 
 ## Upgrade notes
 
-We only support upgrading one major version at a time. Version skipping isn't supported.
+We only support upgrading one major/minor version at a time. Version skipping isn't supported. Please see https://spiffe.io/docs/latest/spire-helm-charts-hardened-about/upgrading/ for details.
+
+### 0.21.X
+
+- If you're doing upgrades as recommended, no special actions should be needed.
```

Comment:
This statement doesn't seem appropriate for upgrade notes, as it effectively says

"Version skipping isn't supported" which is exactly what the line 76 already says.

If you wanted to express some other issue, please be specific in the issue you are discussing, otherwise, consider cutting this line, as it doesn't direct someone to do something, except on a condition that's not clearly mentioned.